environment.yml

```diff
@@ -11,3 +11,4 @@ dependencies:
   - geopandas
   - rasterio
   - scikit-image
+  - gdal = 2
```

Comment:
heya @nkorinek may i ask why this was added here? i ask because i just ran into a weird CI issue with windows and gdal.

Reply:
i think we also may not need this anymore given the most recent CI update. maybe update this from master and see how i t builds. i think you can  then  remove this.